configs/PRINCIPIOTF7/config.h

```diff
@@ -0,0 +1,169 @@
+
+
+// Receiver Configuration
+#define RX_PPM_PIN              PA3 // UART 2 RX
+#define RX_PWM1_PIN             PA2 // UART 2 TX
+#define DEFAULT_RX_FEATURE      FEATURE_RX_SERIAL
+#define SERIALRX_UART           SERIAL_PORT_USART2
+
+
+// I2C/SPI Configuration
+#define I2C1_SCL_PIN            PB8
+#define I2C1_SDA_PIN            PB9
+
+#define SPI1_SCK_PIN            PA5
+#define SPI1_SDI_PIN            PA6
+#define SPI1_SDO_PIN            PA7
+
+#define SPI2_SCK_PIN            PB13
+#define SPI2_SDI_PIN            PB14
+#define SPI2_SDO_PIN            PC3
+
+
+// Sensor Configuration
+#define USE_ACC
+#define USE_GYRO
+#define USE_ACC_SPI_ICM42688P // For ICM42688P variation
+#define USE_GYRO_SPI_ICM42688P // For ICM42688P variation
+
+#define GYRO_1_EXTI_PIN         PC4
+#define GYRO_1_CS_PIN           PB2
+#define GYRO_1_SPI_INSTANCE     SPI1
+#define GYRO_1_ALIGN CW90_DEG
+#define GYRO_1_ALIGN_YAW 1800
+
+#define USE_BARO
+#define USE_BARO_DPS310
+#define BARO_I2C_INSTANCE (I2CDEV_1)
+
+/*
+// SPI barometer for older board revision
+#define USE_BARO_SPI_DPS310
+#define BARO_CS_PIN          PC15
+#define BARO_BUSTYPE SPI
+#define BARO_SPI_INSTANCE SPI2
+#define BARO_HARDWARE           AUTO
+*/
+
+#define MAG_I2C_INSTANCE        (I2CDEV_1)
+
+
+// Flash Configuration
+#define USE_FLASH
+#define USE_FLASH_W25Q128FV
+#define FLASH_CS_PIN            PB12
+#define FLASH_SPI_INSTANCE      SPI2
+#define DEFAULT_BLACKBOX_DEVICE BLACKBOX_DEVICE_FLASH
+
+// DMA configuration
+#define TIMER_PIN_MAPPING \
+    TIMER_PIN_MAP( 0, PC8 , 2,  1) \
+    TIMER_PIN_MAP( 1, PC9 , 2,  0) \
+    TIMER_PIN_MAP( 2, PB4 , 1,  0) \
+    TIMER_PIN_MAP( 3, PB5 , 1,  0) \
+    TIMER_PIN_MAP( 4, PB0 , 2,  0) \
+    TIMER_PIN_MAP( 5, PB1 , 2,  0) \
+    TIMER_PIN_MAP( 6, PB10, 1,  0) \
+    TIMER_PIN_MAP( 7, PB11, 1,  1) \
+    TIMER_PIN_MAP( 8, PB6 , 1,  -1) \
+    TIMER_PIN_MAP( 9, PB7 , 1,  -1) \
+    TIMER_PIN_MAP(10, PA8 , 1,  2) \
+    TIMER_PIN_MAP(11, PA3 , 3, -1) \
+    TIMER_PIN_MAP(12, PA2 , 2,  -1)
+
+#define DEFAULT_DSHOT_BURST DSHOT_DMAR_OFF
+#define DEFAULT_DSHOT_BITBANG DSHOT_BITBANG_ON
+
+
+// ADC Pins
+#define ADC_INSTANCE            ADC1  // Default added
+#define ADC1_DMA_OPT            0  // DMA 2 Stream 0 Channel 0
+
+#define ADC_VBAT_PIN            PC2
+#define ADC_CURR_PIN            PC1
+
+#define DEFAULT_VOLTAGE_METER_SOURCE VOLTAGE_METER_ADC
+#define DEFAULT_CURRENT_METER_SOURCE CURRENT_METER_ADC
+#define DEFAULT_CURRENT_METER_SCALE 179 // This depends on the ESC voltage divider
+
+
+// PINIO Configuration
+#define PINIO1_PIN PC15 // Accessory PINIO 1
+#define PINIO2_PIN PB3 // Accessory PINIO 2
+#define PINIO3_PIN PA15 // Pit mode VTX
+
+#define PINIO1_CONFIG 1 // PINIO_CONFIG_MODE_OUT_PP
+#define PINIO2_CONFIG 1 // PINIO_CONFIG_MODE_OUT_PP
+#define PINIO3_CONFIG 129 // PINIO_CONFIG_MODE_OUT_PP, 10V enable, active low
+
+#define PINIO1_BOX 40 // User 1
+#define PINIO2_BOX 41 // User 2
+#define PINIO3_BOX 39 // Pit mode
```

Comment:
This is not the correct use of pit mode. Since this turns the VTX on and off, you want to set it to a user box. 
https://betaflight.com/docs/wiki/guides/current/Pinio-and-PinioBox#switch-vtx-onoff-using-a-usern-function-controlled-by-the-sender

```suggestion
#define PINIO3_BOX 42 // Pit mode
```

Reply:
Resolved in 3bcb8a6 also updated comment to match